backend/lang/en.json

```
@@ -4817,10 +4827,71 @@
+    "Pallet Movements": "Pallet Movements",
+    "No pallet movements recorded yet.": "No pallet movements recorded yet.",
+    "Logistics operator (can move pallets)": "Logistics operator (can move pallets)",
+    "Select an active logistics operator.": "Select an active logistics operator.",
+    "Pallet movement recorded.": "Pallet movement recorded.",
+    "Select a movable (not shipped) pallet.": "Select a movable (not shipped) pallet.",
     "Bills of Materials": "Bills of Materials",
     "Select one or more BOMs. Requirements sum across the selected BOMs. Leave empty to auto-use the active BOM for the product type.": "Select one or more BOMs. Requirements sum across the selected BOMs. Leave empty to auto-use the active BOM for the product type.",
     "Select a product type first.": "Select a product type first.",
     "No BOMs are available for the selected product type.": "No BOMs are available for the selected product type.",
     "Age": "Age",
+    "just now": "just now",
```

Biome (2.5.3)

[error] The key just now was already declared.

(lint/suspicious/noDuplicateObjectKeys)